pkg/controller/operator.go

```diff
@@ -77,6 +79,27 @@ func (s *Support) Run(controller *controllercmd.ControllerContext) error {
 	if len(s.Impersonate) > 0 {
 		gatherKubeConfig.Impersonate.UserName = s.Impersonate
 	}
+
+	// the metrics client will connect to prometheus and scrape a small set of metrics
+	// TODO: the oauth-proxy and delegating authorizer do not support Impersonate-User,
+	//   so we do not impersonate gather
+	metricsGatherKubeConfig := rest.CopyConfig(controller.KubeConfig)
+	metricsGatherKubeConfig.CAFile = "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
```

Comment:
making this const in `pkg/cmd/start/start.go` but hardcoding here seems strange

Reply:
Consts leaking across packages is bad.  In that case it was just referencing the same variable twice.  Here, eh.